src/config/src/meta/dashboards/mod.rs

```diff
@@ -166,25 +166,13 @@ impl Dashboard {
     }
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize, ToSchema)]
-pub struct Dashboards {
-    pub dashboards: Vec<Dashboard>,
-}
-
 pub mod reports;
 pub mod v1;
 pub mod v2;
 pub mod v3;
 pub mod v4;
 pub mod v5;
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, ToSchema)]
-#[serde(rename_all = "camelCase")]
-pub struct MoveDashboard {
-    pub from: String,
-    pub to: String,
-}
-
```

Comment:
Moved into the `http` module since this is just used as the HTTP request body schema for the `MoveDashboard` route